IFC4x3/Properties/t/Type_2Jl9BkNKP9JB7uUr1oz7oL/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Type_2Jl9BkNKP9JB7uUr1oz7oL" Name="Type" UniqueId="93bc92ee-5d46-494c-b1f8-7b5072f47c95" PropertyType="p_enumeratedvalue" />
```

Comment:
Needs a more specific name.

Reply:
Where is the enumeration?

Reply:
Object Typing is enought. I have erased this property. Commit : 305806220fb93c9010687f137214b3f48df310cb